modules/ROOT/pages/mock-event-processor.adoc

```diff
@@ -45,9 +45,11 @@ This DataWeave file creates the payload to send in a POST request. You can then
 </munit-tools:then-return>
 ----
 
-Finally, you can configure a *then-return* element to define the type of response the mocked processor should return. It could be a payload, a variable, a list of attributes, or even an error.
+Finally, you can configure a *then-return* element to define the type of response the mocked processor returns. It can be a payload, a variable, a list of attributes, or an error.
 
-You can use either a *then-return* to return a static, constant value, or you can use a *then-call* to invoke a flow for cases where you might want the returned value to change over time, or to have different responses according to certain inputs.
+You can use either a *then-return* to return a static, constant value, or you can use a *then-call* to invoke a flow for cases where the returned value can change over time, or to have different responses according to certain inputs.
+
+The mocking of the processors is executed after the initialization of the Mule app, and some connectors can require credentials or configurations to start the app, even when its processors are mocked.
```

Comment:
I would put this as a note, because it's just a clarification about a problem that you might find in some specific cases

Reply:
OK! I like that idea. Thanks!